### ISSUE TITLE
extract database servers to separate containers

### ISSUE THREAD

**Issue by giladkhen** [Metaswitch/clearwater-docker#4]
need in order to support large scale deployment

**Comment by mjbright** [comment]
I'd hoped to use ClearWater vIMS for an Orchestration Lab I'll be running at ContainerCon Europe, but it looks like this isn't scalable at all today.

**Comment by MatMeredith** [comment]
@mjbright We're looking at making some enhancements to clearwater-docker over the next couple of weeks -- if you can provide any more detail on exactly what you'd like to see that would be great input here.   Out of interest, are you still planning to use Clearwater at ContainerCon?

**Comment by mjbright** [comment]
Thanks for the response.

In fact I'm in the final throes of preparation for a container
orchestration lab next week, but I thought of this too late - I wouldn't be
ready to do a scaled version of Clearwater for then.

However, if I manage it I'll use clearwater-docker in it's current form.
I'll keep you informed if I do use it as I would like to go farther in a
future lab.

In this lab the focus will be on using different container orchestration
engines, so this time round I might use ClearWater as a "single instance"
and then connect from container edge nodes.  I'd  be limited to some SIP
command-line client though, I hope I could use SIPP for that - any advice
on that would be appreciated.

**Comment by MatMeredith** [comment]
If all you want to do is to drive small numbers of SIP messages through Clearwater / set-up a simple call from the command line then could you just run e.g. the "Basic Call - Mainline" test from https://github.com/Metaswitch/clearwater-live-test?   This may well be the easiest option?

**Comment by mjbright** [comment]
Thanks a lot Mat, I'll have a look at that (I see it uses sipp also which
is good for me - useful to have a "validated procedure" for that).
That'll help a lot.

**Comment by MatMeredith** [comment]
This is now fixed in #29.  There are associated changes required to a number of other Clearwater repos -- these will be in release-108 (or if you want to start using this now, just update your Base Dockerfile to point to the latest repo rather than stable).   

As well as separating out the data stores, we've also ensured that it is possible to scale up the data store clusters.   See the updated [readme](https://github.com/Metaswitch/clearwater-docker).
